### DISCUSSION TITLE
Locked out of account - how to solve?

### DISCUSSION THREAD

**Discussion by beld78** [community/community#35421] [New to GitHub]
I have a 2FA account without recovery codes and I can't login anymore. When I try to login, I receive this error message: 
![image](https://user-images.githubusercontent.com/94924672/194533002-5874e0f5-32f1-4186-9b56-7505866e9d7a.png)

Now, I checked the documentation on how to solve this. However, the documentation show a different screen with the option "Use a recovery code or request a reset." which I don't have:
![image](https://user-images.githubusercontent.com/94924672/194533262-6fc0f66f-4d8d-4c05-99dd-01611554929f.png)

Is there any way to solve this? I already contacted the support but I need access soon for a project.

**Comment by DSDmark** [comment]
i think it help you
you can use by **Authenticating with a verified device, SSH token, or personal access token**
as describe  
try this 


https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Reply by beld78** [reply]
I followed this page but as you can see the screenshot from the documentation is different from what I see. So I can't follow that.

**Reply by DSDmark** [reply]
### You have Personal access token. ?

As I can see. Did you try all step as in document describe? I focus on 
> authenticating with a verified device, SSH token, or personal access token if you have.

read that section, it may help

**Reply by beld78** [reply]
Yes, but the guide does not say anything about using that. This is what I see when I follow the steps:
![image](https://user-images.githubusercontent.com/94924672/194540475-ac8cdffd-06eb-4a1e-be08-79e93803963b.png)
And when I try to reset my password, as the prompt asks me, this is what I see:
![image](https://user-images.githubusercontent.com/94924672/194540667-290de102-b6bc-487c-919f-b951efef8ab3.png)

As I said in my original post, there is no option to click "Use a recovery code or request a reset.", which is the second step in the guide.

**Reply by DSDmark** [reply]
Then you have nothing left except to wait of email 
 
> Did you add your verification token to GitHub Support? It may fast you verification, I guess 

hope you get your account soon

**Reply by beld78** [reply]
Thanks, I added the token as a reply. I'll have to wait then.

**Reply by Hameed-ipayroll** [reply]
Did they write to you and help you out sorting this issue ? I am going through the same

**Reply by yeypuchii** [reply]
Following, I'm going through the same

**Reply by kequach** [reply]
Yes, someone from the support replied to me after a while. Nothing you can do except wait if you have already contacted them
